draft-ietf-tls-extended-key-update.md

```diff
@@ -762,6 +762,21 @@ used as the "Secret" in the exporter function, defined in
 {{Section 7.5 of TLS}}, to generate EKM, ensuring that
 the exported keying material is aligned with the updated security context.
 
+When the exporter master secret is updated following a successful Extended Key Update,
```

Comment:
```suggestion
When the exporter main secret is updated following a successful Extended Key Update,
```

Reply:
Give the other PR that changes the term from "master" to "main" I was wondering whether we should also change it here.

Reply:
Yes, https://datatracker.ietf.org/doc/draft-ietf-tls-rfc8446bis/ uses "main secret" instead of "master secret".